Part 2: Add API

lmunozm1702 opened the pull request:
## This deliverable includes:
- Add API to save the data filled in the form.
- Get the data on the load event for main windows and for the refresh button at the top of the list section.

### All with arrow functions instead of named functions, and using async/await as asked.

lmunozm1702 committed:
Add API

- Add API to save the data filled in the form.

- Get the data on load event for main windows and for refresh button at the top of the list section.

lmunozm1702 committed:
Fix linter errors.

lmunozm1702 committed:
Removed node_modules folder"

rloterh reviewed (changes requested):
Hi Luis, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- App is running as intended
- Promise feature properly implemented

## Required Changes ♻️
- [x] This activity requires that you use the arrow functions for defining functions/methods. Kindly ensure to improve upon the same.

![image](https://user-images.githubusercontent.com/12745474/202262483-551a3385-04c1-4758-a2de-de49ba9fc39e.png)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

**You can also consider:**
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor. 
Implementing the above change(s) will make your ReadMe/ repository stand out even better 😉

Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

lmunozm1702 committed:
Fix Code review issues.

Fix:

- In render.js refactor using arrow functions.

- In README.md singularize the Authors title to Author.

lmunozm1702 commented:
Hi! @rloterh both issues are solved. Thanks a lot for your review.

lmunozm1702 committed:
Fix linter error.

rloterh reviewed (approved):
## Status: Approved :heavy_check_mark:

Hi Luis,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉💯🌟 

## Highlights 👍
- Nice job implementing recommended changes
- Good ReadMe

Keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
